packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -529,19 +535,19 @@ export class SpecBuilder {
   expectActiveTargets(
     ...targets: Array<{
       query: Query;
-      resumeToken?: string;
-      readTime?: TestSnapshotVersion;
+      resume?: { resumeToken?: string; readTime?: TestSnapshotVersion };
```

Comment:
Why did you combine the `resumeToken` and `readTime` into their own, dedicated object? I may be missing something, but I don't see the purpose in this and it adds a lot of seemingly-unnecessary changes to this PR's diff.